testing/e2e/process_test.go

```diff
@@ -355,3 +351,21 @@ func (s *sensuCtl) run(args ...string) ([]byte, error) {
 func (s *sensuCtl) SetStdin(r io.Reader) {
 	s.stdin = r
 }
+
+// Terminate a Process by sending SIGTERM and waiting for it to exit cleanly.
+// On Windows, there is no way to ask a program to exit cleanly. Instead, it
+// must be killed with SIGKILL.
+func terminate_helper(p *os.Process) error {
```

Comment:
I'd suggest using came case here per Go convention: `terminateHelper`; could also be named `terminateProcess`. Otherwise, I think this is the right place for this function to go 👍

Reply:
+1 to `terminateProcess`

Reply:
Ah of course! I was so focused on the `helper` part of the name that I completely missed the fact that I was using snake case 🤦‍♂️ `terminateProcess` sounds good, I'll update.